couler/core/utils.py

```diff
@@ -255,3 +255,8 @@ def _get_uuid():
 
     """use uuid4 to gen ascii uuid, length=8"""
     return "".join(str(uuid.uuid4()).split("-"))[:8]
+def bool_to_str(b):
+    if b:
+        return 'true'
```

Comment:
Optimize to use one-liner?

Reply:
@terrytangyuan  Thank you so much for your help.  These are few changes I have made. Feel free to give me any advice.
I still have trouble to set up environment for the test, I will figure it out tomorrow and check out any other places that need this util function as well.